test/integration/sequelize.test.js

```diff
@@ -462,15 +462,31 @@ describe(Support.getTestDialectTeaser('Sequelize'), () => {
               'FATAL:  role "bar" does not exist',
               'password authentication failed for user "bar"',
             ].some(fragment => error.message.includes(fragment)));
-          } else if (dialect === 'mssql') {
-            expect(error.message).to.include('Login failed for user \'bar\'.');
-          } else if (dialect === 'db2') {
-            expect(error.message).to.include('A communication error has been detected');
-          } else if (dialect === 'ibmi') {
-            expect(error.message).to.equal('[odbc] Error connecting to the database');
-            expect(error.original.odbcErrors[0].message).to.include('Data source name not found and no default driver specified');
           } else {
-            expect(error.message.toString()).to.match(/.*Access denied.*/);
+            switch (dialect) {
```

Comment:
There is no `postgres-native` for `dialect` so we can use the switch for all dialects. See https://github.com/sequelize/sequelize/blob/344c404eac7393fd5d460cb65eca835c764763d2/test/support.ts#L253-L265

Reply:
Just adding two extra spaces at the start so the indentation of the exepct is the same as the comment

Reply:
Done